memory/ndctl.py

```diff
@@ -129,6 +131,44 @@ def build_fio(self):
                 return os.path.join(sourcedir, "fio")
         return pkg
 
+    def autotools_build_system(self):
+        # Check if /usr/include/iniparser directory is present or not
+        # If not present then create it and then  copy the iniparser.h and dictionary.h headers to /usr/include/iniparser/
+
+        iniparser_dir = "/usr/include/iniparser"
+        if not os.path.exists(iniparser_dir):
+            os.makedirs(iniparser_dir)
+    
+        process.run('cp /usr/include/{iniparser.h,dictionary.h} ' + iniparser_dir, sudo= True, shell=True)
+        process.run('./autogen.sh', sudo=True, shell=True)
+        process.run("./configure CFLAGS='-g -O2' --prefix=/usr "
+                        "--disable-docs "
+                        "--sysconfdir=/etc --libdir="
+                        "/usr/lib64", shell=True, sudo=True)
+        build.make(".")
+
+        self.ndctl = os.path.abspath('./ndctl/ndctl')
+        self.daxctl = os.path.abspath('./daxctl/daxctl')
+
+    def meson_build_system(self, deps):
+        if self.dist.name == 'SuSE':
+            deps.extend(['xmlto', 'libgudev-1_0-devel', 'libuuid-devel', 'libiniparser-devel', 'libiniparser1', 'ruby2.5-rubygem-asciidoctor-doc', 'systemd-rpm-macros', 'pkg-config', 'meson', 'cmake'])
+        elif self.dist.name == 'rhel':
+            # Skipping this for now, due to non-availibility of dependednt packages.
```

Comment:
Please cancel the test in this case

Reply:
I am cancelling with the following message - 

**_Cancelling the test due to the non-availability of dependent packages._**

Please confirm.

Reply:
yes

Reply:
Thanks